egs2/TEMPLATE/asr1/db.sh

```diff
@@ -1,20 +1,16 @@
 # Set the path of your corpus
 # "downloads" means the corpus can be downloaded by the recipe automatically
 
-ACCENTED_FR=downloads
```

Comment:
could you not remove the entries for other datasets?

Reply:
For now, I will remove the automatic download.

Reply:
No, please do not remove other databases from `db.sh`.
You may restore by 
```
git fetch origin
git checkout origin/master -- egs2/TEMPLATE/asr1/db.sh
# then add your new DB and commit it
...
```